drivers/gpio/gpio_mcux_lpc.c

```diff
@@ -296,6 +296,7 @@ static int gpio_mcux_lpc_pin_interrupt_configure(struct device *dev,
 		data->pint_id[pin] = pint_idx;
 	}
 
+	PINT_Init(config->pint_base);
```

Comment:
Move this to soc.c like lpc55s69

Reply:
Since PINT is used only in this file, wouldn't this be a place to put this instead of soc.c?

Reply:
@mmahadevan108 PINT initialization is done at soc level as it's serving multiple gpio device 0 and 1, you may end up initializing it twice, with no side effect; also it's similar to clock setup done in soc.c, clocks setup and hw block initialization serving other devices.

Reply:
This call is not SoC specific like the clocking setup. I would be duplicating the same line in all soc files, also it leaves the chance for future soc additions to miss calling this function.
I can add a flag check so this gets initialized once.